src/routes/viewRoutes.js

```diff
@@ -0,0 +1,9 @@
+const { Router } = require("express");
+const { renderHome, renderAllNotes } = require("../controllers/view");
+
+const router = Router();
+
+router.get("/", renderHome);
```

Comment:
Currently if we go to any route that isn't just `/` or `/notes` your application breaks. I can type `/home` in the URL and it will throw an error saying `Cannot GET /home`. We need to try and protect against this, and as suggested in the homework README, we should use `*` to mean anything which follows the `/`. So here we would have:

```
router.get("*", renderHome);
```

Doing this before the `/notes` route though would mean the `/notes` is never reached, so we would need to switch the order of the routes.